styx-scheduler-service/src/main/java/com/spotify/styx/docker/KubernetesDockerRunner.java

```diff
@@ -157,6 +162,38 @@ public String start(WorkflowInstance workflowInstance, RunSpec runSpec) throws I
     }
   }
 
+  @Override
```

Comment:
I'm curious if peoples behaviour is to remove or disable workflows when not using them. Because if the latter is the case, we wouldn't have real benefit..